src/puya/awst_build/eb/arc4/abi_call.py

```diff
@@ -1,6 +1,8 @@
+from __future__ import annotations
```

Comment:
I've been removing this expect where it's required - doesn't look like it is here?